src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.cs

```diff
@@ -30,7 +30,7 @@ public partial struct CounterSample
         public static bool operator ==(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
         public static bool operator !=(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
     }
-    public sealed partial class PerformanceCounter
+    public sealed partial class PerformanceCounter : System.ComponentModel.Component, System.ComponentModel.ISupportInitialize
```

Comment:
We have no idea. We can delete it, if @brianrob, @vancem, @valenis wish.

Apparently it's harmless as it can't get in the package as it has no impl, and can't successfully get referenced by something in the package.